packages/flutter/lib/src/widgets/default_text_editing_shortcuts.dart

```diff
@@ -224,7 +224,45 @@ class DefaultTextEditingShortcuts extends Shortcuts {
   //   * Meta + backspace
   static const Map<ShortcutActivator, Intent> _androidShortcuts = _commonShortcuts;
 
-  static const Map<ShortcutActivator, Intent> _fuchsiaShortcuts = _androidShortcuts;
+  // Fuchsia keyboard HID usage values are defined in (page 53):
```

Comment:
Why mention HID usage here? Do you mean Fuchsia uses HID as their logical key value? If so you should document it that way (because normally OSs uses different, platform-dependent values for logical keys.)

Reply:
I just copied the comment over from some fuchsia code that overrides the shortcuts to get text editing shortcuts to work on their end. But from the looks of it yes for instance <kbd>backspace</kbd> is defined as `LogicalKeyboardKey(0x2a | fuchsiaPlane)`, the `fuchsiaPlane` mask is applied in the embedder I think? Anyways I don't have more information than that if you think the comment should be removed I'll remove that.

Reply:
Looks like I made a mistake when porting Fuchsia keyboard to input3.

See here for details: https://bugs.fuchsia.dev/p/fuchsia/issues/detail?id=88771#c3

Sorry fort the kerfuffle - it seems that it should be fairly easy to fix.